js/modal.js

```diff
@@ -0,0 +1,134 @@
+closeModal.addEventListener('click', () => modal.classList.add('hidden'));
+
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'tonic1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'multi-post1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'tonic2.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'multi-post2.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+];
+
+projects.forEach((project) => {
+  const article = document.createElement('article');
+  article.classList.add('work-card');
+  function concatenateTechnologies(acc, cur) {
+    acc += `<li class="tag">${cur}</li>`;
+    return acc;
+  }
+
+  const technologies = project.technologies.reduce(concatenateTechnologies, '');
+  const buttonId = `see-project-${project.id}`;
+  article.innerHTML = `
+  <div class="work-card_picture">
+    <img src="img/Posts/${project.featuredImage}" alt="Tonic" />
+  </div>
```

Comment:
- [ ] **[Optional]** I think it would be better to resize the window to match the Figma design, you may try to give the same margin for the top, left, right, and bottom  👍 👍